rasa/multi_skill.py

```diff
@@ -0,0 +1,148 @@
+import logging
+from typing import Text, List, Union, Set, Dict, Optional
+import os
+
+import rasa.utils.io as io_utils
+from rasa import data
+
+logger = logging.getLogger(__name__)
+
+
+class SkillSelector:
+    def __init__(self, imports: Set[Text], project_directory: Optional[Text] = None):
+        self._imports = imports
+        self._project_directory = project_directory
+
+    @classmethod
+    def empty(cls) -> "SkillSelector":
+        return cls(set())
```

Comment:
why is the `projecty_directory` not set in this case? would it not be better to always set it instead of having it as an optional?

Reply:
I had it set to the current working directory but this got really ugly when running the tests since their are all sorts of other md files which then become import candidates in the project root